addons/im_livechat/static/src/core/attachment_panel_patch.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.AttachmentPanel" t-inherit-mode="extension">
+        <xpath expr="//ActionPanel//*" position="before">
+            <div t-if="props.thread.type === 'livechat'" class="form-check form-switch">
+                <input t-on-change="toggleAllowVisitorUpload" class="form-check-input" type="checkbox" role="switch" id="im_livechat.allow_upload" t-att-checked="props.thread.channel.allow_visitor_upload"/>
+                <label class="form-check-label" for="im_livechat.allow_upload" t-esc="toggleAllowVisitorUploadText "/>
```

Comment:
The `id` should be unique per livechat. You could have several chat windows with livechat that show the attachment panel

Reply:
Good catch, thanks.